spec/javascripts/Dashboard/chart.spec.js

```diff
@@ -1,5 +1,5 @@
 import $ from 'jquery'
-import {render} from 'dashboard/chart'
+import {render} from '../../../app/javascript/src/Dashboard/chart'
```

Comment:
There has to be some way of getting rid of this and adding second "root" or something.

Reply:
I'll keep looking how to do it with karma-webpack. It wasn't as straightforward as with `jest`